examples/direct-upload/test/server.test.js

```diff
@@ -0,0 +1,26 @@
+var test = require('tape')
+var server = require('../lib/index.js')
+
+test('simple server running', function (t) {
+  var options = {
+    method: 'GET',
+    url: '/'
+  }
+
+  server.inject(options, function (response) {
```

Comment:
Add server.init function here and inside callback access server.inject for testing, see this example

https://github.com/davethomas11/stackoverlfow_Q_39432656/blob/master/test/index.js#L24